lib/google_contacts_api/contact.rb

```diff
@@ -1,19 +1,36 @@
 module GoogleContactsApi
+
```

Comment:
I've mostly updated the contacts and group class to have all getters and setters needed, however I haven't written any new tests for my code here, and some of it still isn't very DRY.